### PR TITLE
Update Polyfill in getting-started

### PR DESCRIPTION
**Pull request by rmi22186**
changed 'which' to 'with'

...providing JavaScript which tests for features...

to:

providing JavaScript with tests for features..

### CODE REVIEW

**Comment by tomdale** [comment]
I don't think this is incorrect usage, but could probably be revised to be clearer:

> This is a term that describes code that detects features that are missing in the user's browser and "fills" them by providing an implementation.

We may want to provide a link to something like https://remysharp.com/2010/10/08/what-is-a-polyfill as well.

File: source/getting-started/glossary.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ A validation tool which checks for common issues in your JavaScript. You'd usual
 
 
 ## Polyfill
-This is a concept that typically means providing JavaScript which tests for features that are missing (prototypes not defined, etc) and "fills" them by providing an implementation.
+This is a concept that typically means providing JavaScript with tests for features that are missing (prototypes not defined, etc) and "fills" them by providing an implementation.
```
Comment (locks) [inline source/getting-started/glossary.md]: `that tests for features` would be a better alternative. `with tests` changes the meaning of the sentence.

**Comment by locks** [comment]
@rmi22186 can you add the link mentioned?

File: source/getting-started/glossary.md (hunk 1 above)
Comment (locks) [inline source/getting-started/glossary.md]: another suggestion: `This is a concept that typically means using JavaScript to test for features`

**Comment by locks** [comment]
@rmi22186 poke, can you address my comments and tom's suggestion?

**Comment by michaelrkn** [comment]
I'm going to close this out. If somebody wants to move it forward, feel free to address the above comments and re-open or open a new PR. Thanks!

**Pull request closed without merging**